apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -88,19 +86,21 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
   Scaffold(
     topBar = {
       if (!oneHandUI.state.value) {
-        Box(Modifier.padding(end = endPadding)) {
+        Column(Modifier.padding(end = endPadding)) {
           ChatListToolbar(
             scaffoldState.drawerState,
             userPickerState,
             stopped,
             oneHandUI
           )
+          Divider()
         }
       }
     },
     bottomBar = {
       if (oneHandUI.state.value) {
-        Box(Modifier.padding(end = endPadding)) {
+        Column(Modifier.padding(end = endPadding)) {
+          Divider()
           ChatListToolbar(
```

Comment:
There is a 1.dp difference in chat list bottom toolbar and new chat sheet bottom toolbar (new chat sheet is smaller in height)